core/tests/protractor.conf.js

```diff
@@ -166,7 +166,7 @@ exports.config = {
   // find chromedriver. This will be passed to the selenium jar as
   // the system property webdriver.chrome.driver. If null, selenium will
   // attempt to find chromedriver using PATH.
-  chromeDriver: './selenium/chromedriver',
+  chromeDriver: null,
```

Comment:
Why this change?

Reply:
This was done so that the chromedriver version is fetched from the PATH instead of hard-coding it since a version upgrade would download a file with the version name suffix. This is not necessary now since this PR no longer requires this update.